docs/src/3.4.MouseEvents.mdx

```diff
@@ -75,3 +75,7 @@ Check out below example for automatic/custom hitmap mapping, and regular/instanc
 ## Full Example
 
 <MouseEvents />
+
+## Other
+
+We use `mouseClickRadius` prop to determine if mouseClick event is going to be triggered, specificially if the mouse has moved beyond the `mouseClickRadius` limit, the `onClick` event handler will be canceled, and `onMouseUp` will still get triggered if present.
```

Comment:
probably best to also add a description of the common use case, i.e. this helps implement the desired behavior of dragging the camera around not firing `onClick` events

Reply:
Good idea! Will add